daemon/config/README.md

```diff
@@ -131,7 +132,8 @@ To understand what this section is and how it should be configured, please refer
 
 ### [backend]
 
-Refer to the [streaming and replication](/streaming/README.md) documentation.
+The backends system is deprecated. Refer to [exporting](/exporting/README.md) for details on sending metrics to external
```

Comment:
Hello @joelhans , the system name is `backend`, why did you write a plural here?

Reply:
Good question! I'm not sure why I did, but it's removed now.